aspnetcore/host-and-deploy/iis/troubleshoot.md

```diff
@@ -15,46 +15,140 @@ uid: host-and-deploy/iis/troubleshoot
+
+**Framework-dependent deployment**
+
+If the app is a [framework-dependent deployment](/dotnet/core/deploying/#framework-dependent-deployments-fdd):
+
+1. At a command prompt, set the `ASPNETCORE_DETAILEDERRORS` environment variable. Use the command: `set ASPNETCORE_DETAILEDERRORS=true`.
+1. Navigate to the deployment folder and run the app by executing the app's assembly with *dotnet.exe*. In the following command, substitute the name of the app's assembly for \<assembly_name>: `dotnet .\<assembly_name>.dll`.
+1. The console output from the app, showing any errors, is written to the console window.
+1. If the errors occur when making a request to the app, make a request to the host and port where Kestrel listens. Using the default host and post, make a request to `http://localhost:5000/`. If the app responds normally at the Kestrel endpoint address, the problem is more likely related to the reverse proxy configuration and less likely within the app.
+1. When finished, remove the `ASPNETCORE_DETAILEDERRORS` environment variable. Use the command: `set ASPNETCORE_DETAILEDERRORS=`.
+
+**Self-contained deployment**
+
+If the app is a [self-contained deployment](/dotnet/core/deploying/#self-contained-deployments-scd):
+
+1. At a command prompt, set the `ASPNETCORE_DETAILEDERRORS` environment variable. Use the command: `set ASPNETCORE_DETAILEDERRORS=true`.
+1. Navigate to the deployment folder and run the app's executable. In the following command, substitute the name of the app's assembly for \<assembly_name>: `<assembly_name>.exe`.
+1. The console output from the app, showing any errors, is written to the console window.
+1. If the errors occur when making a request to the app, make a request to the host and port where Kestrel listens. Using the default host and post, make a request to `http://localhost:5000/`. If the app responds normally at the Kestrel endpoint address, the problem is more likely related to the reverse proxy configuration and less likely within the app.
+1. When finished, remove the `ASPNETCORE_DETAILEDERRORS` environment variable. Use the command: `set ASPNETCORE_DETAILEDERRORS=`.
+
+### ASP.NET Core Module stdout log
+
+To enable and view stdout logs:
+
+1. Navigate to the site's deployment folder on the hosting system.
+1. If the *logs* folder isn't present, create the folder. For instructions on how to enable MSBuild to create the *logs* folder in the deployment automatically, see the [Directory structure](xref:host-and-deploy/directory-structure) topic.
+1. Edit the *web.config* file. Set **stdoutLogEnabled** to `true` and change the **stdoutLogFile** path to point to the *logs* folder (for example, `.\logs\stdout`). A timestamp, process id, and file extension are added automatically when the log is created (for example, *stdout_20180205184032_5412.log*).
```

Comment:
The .\logs\stdout confuses people as they think stdout must be a folder. It's worth mentioning it's the log file prefix.